### ISSUE TITLE
Failed S/R with LanguageDepotProgrammer role

### ISSUE THREAD

**Issue by megahirt** [sillsdev/web-languagedepot#15]
This is a LANGUAGE DEPOT issue.

Recently during testing S/R with a project, @rmunn and I encountered an issue where the "push" step of the LD S/R fails if the user's redmine role is `LanguageDepotProgrammer`.  This used to work, but now it consistently fails.  I'm not sure what the issue is.

**Comment by rmunn** [comment]
I believe this may have been failing on private projects. I just logged into Language Depot and found that although the **public** instance of Redmine has given "Commit access" permissions to the LanguageDepotProgrammer role, the LanguageDepotProgrammer role in the **private** instance of Redmine was lacking "Commit access" permissions. "Commit access" permissions is what Redmine uses to allow hg push, while hg pull is allowed if the user has either "Commit access" or "Browse repository" permissions. I've fixed the LDP permissions on the private instance of Redmine, and I believe this issue is fixed. We can reopen it later if we discover that it isn't.
